transcribed/page1160.txt

```diff
@@ -0,0 +1,23 @@
+001             ;*SYSTEM* JOBS                                                  PAGE 116
+002
+003     117 016 SCORE:   SKIPL SUPCOR
+004     017 002          PUSHJ P,UPL3
+005     116 017          PUSHJ P,SCORE4
+006     117 013          CLEARM @SYSL
+007     117 013          HRLX A,SYSL
+008     117 013          HRM A,SYSL
+009     001 006          AOS A
+010     001 007          MOVE I A,2000
+011     117 013          ADD S,SYSL
```

Comment:
ADD B,

Reply:
It is so hard to see. But A and B tend to be frequently used, so you are more than likely correct. Let's take your version and see if we need to fix it later when we get better photos.